### PR TITLE
source snapshots from sonatype only

### PR DESCRIPTION
**Pull request by joshk0**
LiveRamp/pom-common#6 makes this possible by pushing pom-common to
sonatype, this change validates that the change works.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -146,7 +146,7 @@ note: "submits work to a service" technically your waypoint can just do work in
 
 ###### Synchronous Waypoint
 
-This Wayoint submits work to a service, and then, as long as the submission of that work does not fail, moves the request on to the next step in your manifest.
+This Waypoint submits work to a service, and then, as long as the submission of that work does not fail, moves the request on to the next step in your manifest.
```
Comment (h-wang94) [inline README.md]: Thanks 😆

File: .travis.yml [hunk 2]
```diff
@@ -1,3 +1,4 @@
+dist: trusty
```
Comment (h-wang94) [inline .travis.yml]: What is this for?

File: pom.xml [hunk 3]
```diff
@@ -76,13 +76,10 @@
 
   <repositories>
     <repository>
-      <id>all-snapshots</id>
-      <name>Liveramp Repositories</name>
-      <url>http://repository.liveramp.com/artifactory/libs-snapshot-local</url>
```
Comment (h-wang94) [inline pom.xml]: Do users of this package need to update anything?

File: .travis.yml (hunk 2 above)
Comment (joshk0) [reply .travis.yml]: I am not sure. When I did not have this there, the [travis process](https://travis-ci.com/LiveRamp/captain/builds/111505854) errored on setting up JDK8. I could try taking it out again if you think it's not necessary.

File: pom.xml (hunk 3 above)
Comment (joshk0) [reply pom.xml]: This change only updates where the build process for captain downloads its dependencies from. It shouldn't have any impact on users 🤞

File: .travis.yml (hunk 2 above)
Comment (h-wang94) [reply .travis.yml]: Oh i see. Eh, was mostly just wondering why this exists. I'll trust you on this 😆

File: pom.xml (hunk 3 above)
Comment (h-wang94) [reply pom.xml]: Oh, right. This is to tell it where to download from. My bad. Noob mistake

**Pull request merged**